atom/browser/auto_updater_mac.mm

```diff
@@ -22,6 +22,8 @@
 
 }  // namespace
 
+bool update_available = false;
```

Comment:
The global variables should be moved into anonymous namespace and the name should starts with `g_`. The `g_updater` variable is a good example here.